gradyrobbins/fifty
# Clifbar

# Description

upon npm start ,  getting an error in the browser.

fixed via implementing a solution found at: 
```https://stackoverflow.com/questions/60234640/typeerror-err-invalid-arg-type-the-path-argument-must-be-of-type-string-re##``` 

Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made
run through the steps described in readme to verify the app renders in the browser
# Checklist:
